source/js/components/news/news.jsx

```diff
@@ -21,7 +28,7 @@ export default class News extends React.Component {
       });
     });
 
-    xhr.open(`GET`, `https://${env.NETWORK_API_DOMAIN}/api/news/?format=json&featured=True&page=1`);
+    xhr.open(`GET`, `https://${networkApiDomain}/api/news/?format=json&featured=True&page=1`);
```

Comment:
I think @gvn recently refactored this out with the JSX migration. You may not need these changes if that's landed in master. If we do still need the current domain in the front-end, can we make it not baked in at build-time so that we don't run into the "heroku promotion" bug we've got on Learning.m.o and other sites?

(This comment applies to the other JSX components too)

Reply:
> not baked in at build-time
👍

This branch appears to be up to date, but we might consider holding off until @gvn's work lands.

Reply:
I'm done with migrating the CMS driven components to JSX. 

Is there a reason we'd want to pull from the Heroku app's API instead of prod (or whatever NETWORK_API_DOMAIN is potentially overridden to)? Most of the time we're going to want to see the site with prod's data.